code/controllers/Processes/pipenet.dm

```diff
@@ -1,14 +1,16 @@
-var/global/list/datum/pipe_network/pipe_networks = list()
-var/global/list/obj/machinery/atmospherics/atmos_machines = list()
+var/list/datum/pipe_network/pipe_networks = list()
```

Comment:
Cluster I know you're meming but strong typing is more likely to happen than this.